### DISCUSSION TITLE
Autostart fails

### DISCUSSION THREAD

**Discussion by marco-prolo** [MonitorControl/MonitorControl#1561] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

When I select "start on startup" in the options, the app won't start after booting. Although it's listed in the startup settings of  the mac settings, there is no icon, as if the app is nonexistent to the os. See screenshot:
![Bildschirmfoto 2024-05-08 um 08 06 17](https://github.com/MonitorControl/MonitorControl/assets/63793737/10c2aa74-fbd7-45d0-918f-f15201c154e8)

I also manually deleted it from the mac setting and put it back there. Also with no luck. And after the reboot the "start on startup box in the monitor control app is unticked again...." 
I already removed the app via app cleaner and reinstalled with no success to date.   

### Steps to reproduce

Chose "Start on Startup" in the monitor control options

### Expected behavior

It should start when booting into mac os

### Environment Information (please complete the following information)

```markdown
- macOS version: 14.4.1
- Mac model: Mac Studio M2 Max
- MonitorControl version: 4.2.0
- Monitor(s): Lg 40" Ultrawide 5K
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Something does not seem right with the installation. Do you have the app installed under /Applications? Did you try to manually drag the app to Login Items?
